mindjunkies/courses/management/commands/populate_category.py

```
@@ -0,0 +1,48 @@
+import random
```

Ruff (0.8.2)

`random` imported but unused

Remove unused import: `random`

(F401)